nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -283,17 +283,23 @@ int ddxProcessArgument(int argc, char *argv[], int i)
       }
     }
 
-    if (nxagentOptionFile)
+    if (nxagentOptionsFilename)
     {
-      /* if the "filename" starts with nx/nx treat it
-	 as an option string */
-      if (strncasecmp(nxagentOptionFile, "nx/nx", 5) == 0)
+      /* if the "filename" starts with an nx marker treat it
+	 as an option _string_ instead of a filename */
+      if (strncasecmp(nxagentOptionsFilename, "nx/nx,", 6) == 0 ||
+	  strncasecmp(nxagentOptionsFilename, "nx/nx:", 6) == 0 ||
       {
-	nxagentParseOptionString(nxagentOptionFile + 5);
+	nxagentParseOptionString(nxagentOptionsFilename + 6);
+      }
+      else if (strncasecmp(nxagentOptionsFilename, "nx,", 3) == 0 ||
+	       strncasecmp(nxagentOptionsFilename, "nx:", 3) == 0)
+      {
+	nxagentParseOptionString(nxagentOptionsFilename + 3);
       }
       else
       {
-	nxagentProcessOptionsFile(nxagentOptionFile);
+	nxagentProcessOptionsFile(nxagentOptionsFilename);
```

Comment:
Thanks for adding those. However, these changes do not belong into this commit, it seems. Rather into the previous commit. I know this is picky, but I'd like the history to be as clean as possible.

Also: you haven't updated the "addresses <issue>" line the previous commit's commit msg. Please use "Fixes ArcticaProject/nx-libs#<issue>." there.